homeassistant/components/nws/weather.py

```diff
@@ -165,16 +164,18 @@ def convert_condition(time, weather):
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the NWS weather platform."""
 
+    if (config.get(CONF_LATITUDE) and not config.get(CONF_LONGITUDE)) or (
```

Comment:
Latitude and longitude should be inclusive in the config schema. So we won't need this check after that change.